samples/algorithms/integer-factorization/Program.cs

```diff
@@ -142,14 +145,25 @@ static int Estimate(EstimateOptions options)
         }
 
         static int Visualize(VisualizeOptions options) {
-            var config = FlameGraphResourcesEstimator.RecommendedConfig();
+            if (options.QuantumViz) {
+                var config = QuantumVizEstimator.RecommendedConfig();
 
-            var estimator = new FlameGraphResourcesEstimator(config, options.Resource);
+                var estimator = new QuantumVizEstimator(config);
 
-            var bitsize = (long)System.Math.Ceiling(System.Math.Log2(options.NumberToFactor + 1));
-            EstimateFrequency.Run(estimator, options.Generator, options.NumberToFactor, options.UseRobustPhaseEstimation, bitsize).Wait();
+                var bitsize = (long)System.Math.Ceiling(System.Math.Log2(options.NumberToFactor + 1));
```

Comment:
This line can be moved outside of the if statement, it doesn't seem to depend on the path